source/getting-started/installation-vagrant.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: page
+title: "Installation on Vagrant"
+description: "Instructions to install Home Assistant on a Vagrant VM."
+date: 2016-04-19 21:28
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
+
+A Vagrantfile is provided for quickly spinning up an ubuntu vm running home-assistant.
+This can be beneficial for those who want to experiment with home-assistant and/or developers.
+
+
+ ## Starting vagrant
+
+ You must have [vagrant](https://www.vagrantup.com/downloads.html) and [virtualbox](https://www.virtualbox.org/wiki/Downloads) installed.
+
+<p class='note'>
+Vagrant is intended for testing/development only. It is NOT recommended for permanent installations.
+</p>
+
+Download the home-assistant source code by either downloading the .zip file from [github releases page](https://github.com/home-assistant/home-assistant/releases), or by using [git](https://git-scm.com/)
+
+    git clone https://github.com/home-assistant/home-assistant.git
+    vagrant up
+
+This will launch an Ubuntu 14.04 VM using Virtualbox. Home-Assistant will then be available at the following ip:port
+
+http://192.168.33.10:8123
```

Comment:
Yes, I agree that will probably be better. Will add that when back on my normal computer later today.